examples/py/async-binance-create-trailing-percent-order.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+import asyncio
+import os
+import sys
+from pprint import pprint
+
+root = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.append(root + '/python')
+
+import ccxt.async_support as ccxt  # noqa: E402
+
+
+async def main():
+    exchange = ccxt.binanceusdm({
```

Comment:
@Dan-krm next time please try to add the example in ts so we can transpile to all languages